src/pages/trader-tools/common/_style.ts

```diff
@@ -104,7 +104,7 @@ export const SectionSubtitle = styled(Header)`
         padding: 0 16px;
     }
 `
-const swap_tab_selector_css = (props) =>
+const swap_tab_selector_css = (props: { active: boolean }) =>
```

Comment:
replacing props with the parameters that are used in this function is the better approach.

Reply:
Yes hirad that's right, I agree with this and this was one of my recommendations for refactoring codes
For now since this is not under this card's scope I left it like this, but I believe in the near future we will refactor all of them!